src/board_controller/freeeeg32/freeeeg32.cpp

```diff
@@ -202,30 +202,24 @@ int FreeEEG32::open_port ()
 
 int FreeEEG32::set_port_settings ()
 {
-#ifdef _WIN32
-    // windows driver fails to set settings and in fact ignores them, no idea what drivers on others
-    // OSes do
-    int timeout_only = true;
-#else
-    int timeout_only = false;
-#endif
-    int res = serial->set_serial_port_settings (1000, timeout_only);
+    int res = serial->set_serial_port_settings (1000, false);
     if (res < 0)
     {
         safe_logger (spdlog::level::err, "Unable to set port settings, res is {}", res);
         return (int)BrainFlowExitCodes::SET_PORT_ERROR;
```

Comment:
I think this one should be under `#ifndef _WIN32` too

Reply:
Fixed.